src/software/physics/euclidean_to_wheel.cpp

```diff
@@ -0,0 +1,128 @@
+#include "euclidean_to_wheel.h"
+
+#define _USE_MATH_DEFINES
```

Comment:
why's this needed?

Reply:
Lines 21, 22 its used to convert degrees to radians.